rocketmq-spring-boot/src/test/java/org/apache/rocketmq/spring/support/DefaultRocketMQListenerContainerTest.java

```diff
@@ -46,6 +69,180 @@ public void onMessage(MessageExt message) {
         result = (Class) getMessageType.invoke(listenerContainer);
         assertThat(result.getName().equals(MessageExt.class.getName()));
     }
+
+    @Test
+    public void testSelectorType() throws Exception {
+        DefaultRocketMQListenerContainer listenerContainer = new DefaultRocketMQListenerContainer();
+        listenerContainer.setConsumer(new DefaultMQPushConsumer());
+        Method initSelectorType = DefaultRocketMQListenerContainer.class.getDeclaredMethod("initSelectorType");
+        initSelectorType.setAccessible(true);
+
+        try {
+            listenerContainer.setRocketMQMessageListener(TagClass.class.getAnnotation(RocketMQMessageListener.class));
+            initSelectorType.invoke(listenerContainer);
+
+            listenerContainer.setRocketMQMessageListener(SQL92Class.class.getAnnotation(RocketMQMessageListener.class));
+            initSelectorType.invoke(listenerContainer);
+        } catch (Exception e) {
+
+        }
+    }
```

Comment:
Lack of assertion

Reply:
resolve to: https://github.com/apache/rocketmq-spring/pull/205